ml-agents/mlagents/trainers/models.py

```diff
@@ -302,7 +434,9 @@ def create_discrete_action_masking_layer(all_logits, action_masks, action_size):
             ),
         )
 
-    def create_observation_streams(self, num_streams, h_size, num_layers):
+    def create_observation_streams(
```

Comment:
Please add type annotations here. That would have told you that you've got a mismatch between the default value of vis_encode_type (a string) and the comparisons (using the enum).